### DISCUSSION TITLE
How to label or add description to bookmark

### DISCUSSION THREAD

**Discussion by Raynchowkw** [alefragnani/vscode-bookmarks#528] [Q&A] [answered]
Hi,
I see the description of adding label to bookmark, but I don't find a way to do it.
I want to label bookmark, and then find bookmark only for one specific label for one purpose.
Thanks.

**Comment by alefragnani** [comment, marked as answer]
Hi @Raynchowkw ,

You have three ways to do it:

* Use `Bookmarks: Toggle Labeled` command
* In the Side Bar, use the Context Menu command `Edit Label` on each bookmark
* In the Side Bar, use the inline command `Edit Label` (similar to a pencil) on each bookmark

And, about the `Bookmarks: Toggle Labeled` command, you could play with the `bookmarks.label.suggestion` setting. More info in README https://github.com/alefragnani/vscode-bookmarks#available-settings.

To search bookmarks, use the `Bookmarks: List` or `Bookmarks: List from All Files` commands

Hope this helps

**Comment by Raynchowkw** [comment]
Thanks. It helps!
